scripts/mk.va4.js

```diff
@@ -1 +1 @@
ages/items/'+o+'.png" style="height: '+Math.round(iScreenScale*l)+'px;" />':""}var s=document.getElementById("scroller"+e),c=document.getElementById("objet"+e);t.stash?(document.getElementById("reserve"+e).style.visibility="visible",c.style.left=Math.round(3*iScreenScale)+"px",c.style.top=Math.round(3*iScreenScale)+"px",s.style.left=Math.round(3*iScreenScale)+"px",s.style.top=Math.round(3*iScreenScale+ +iScreenScale)+"px"):(document.getElementById("reserve"+e).style.visibility="hidden",c.style.left=Math.round(iScreenScale)+"px",c.style.top=Math.round(iScreenScale)+"px",s.style.left=iScreenScale+"px",s.style.top=Math.round(iScreenScale+ +iScreenScale)+"px")}function updateItemCountdownHud(e,t){var n,a,o,i,r,l=document.getElementById("countdown"+e);l&&(0<t?(l.style.display="block",n=8.5*iScreenScale,a=6.25*iScreenScale,o=Math.atan2(a,n),r=.2+t*((i=2*Math.PI)-.2)+o,e=n/2-n*Math.cos(r),t=a/2-a*Math.sin(r),t=[[0,0],[iScreenScale,0],[Math.round(2.5*iScreenScale),2*iScreenScale],[n/2,a/2],[e,t]],i-o<r&&t.push([0,a]),r>i-Math.PI+o&&t.push([n,a]),r>i-Math.PI-o&&t.push([n,0]),l.style.clipPath="polygon("+t.map(function(e){return e[0]+"px "+e[1]+"px"}).join(", ")):(l.style.display="none",l.style.clipPath=""))}function timeStr(e){var t=Math.floor(e/6e4);e-=6e4*t,t+="";var n=Math.floor(e/1e3);for(e-=1e3*n,(n+="").length<2&&(n="0"+n),e+="";e.length<3;)e="0"+e;return t+"'"+n+"&quot;"+e}function openCheats(){var e=prompt("MKPC Console command");e&&(processCode(e)?(clLocalVars.cheated=!0,clSelected&&challengeHandleFail()):alert("Invalid command"))}function processCode(e){if("/"==e.charAt(0)){e=e.substring(1);var t=oPlayers[0],n=/^give (\w+)$/g.exec(e);if(n){for(var a=n[1],o=!1,i=0;i<itemDistribution.length;i++)if(itemDistribution[i][a]){o=!0;break}return o?(t.billball?(t.stash=a,t.roulette2=25):(t.arme=a,t.roulette=25),updateObjHud(0),1):void 0}n=/^tp ([\d\-+\.]+) ([\d\-+\.]+)$/g.exec(e);if(n=n||/^tp ([\d\-+\.]+) ([\d\-+\.]+) ([\d\-+\.]+)$/g.exec(e)){var r=parseFloat(n[1]),l=parseFloat(n[2]);if(isNaN(r)||isNaN(l))return;var s=parseFloat(n[3]);return t.x=r,t.y=l,isNaN(s)||(t.rotation=s),resetRenderState(),1}r=/^lap(?: ([1-3]|c))?(?: (\d+|c))?$/g.exec(e);if(r){l=parseInt(r[1]),s=parseInt(r[2]);if("c"==r[1]&&(l=t.tours),r[1]||(l=oMap.tours,s=oMap.checkpoint.length-1),"c"==r[2]&&(s=t.demitours),r[2]||(s=oMap.checkpoint.length-1),isNaN(l)||isNaN(s))return;t.tours=l,t.demitours=s;for(var c=document.querySelectorAll("#compteur0 .tour"),i=0;i<c.length;i++)c[i].innerHTML=t.tours;return 1}if("pos"==e)return alert("x: "+Math.round(t.x)+"\ny: "+Math.round(t.y)+"\ntheta: "+Math.round(t.rotation)),1;if("xs"==e)return t.size=.6,t.mini=0,1;if("md"==e)return t.size=1,t.mini=0,1;if("xl"==e)return t.size=Math.pow(1.05,8),t.mini=0,1;if("BB"==course){"balloon"==e&&(e+=" 1");e=/^balloon (\d+)$/g.exec(e);if(e){e=parseInt(e[1]);if(e)return t.reserve+=e,updateBalloonHud(document.getElementById("compteur0"),t),1}}}}function distToAiPoints(e,t,n){for(var a=1/0,o=0;o<n.length;o++){var i=(o+1
```

Comment:
You don't need to change those files, I usually auto-generate them when I make a release